lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1115,10 +1155,10 @@ instance LiftHandler ErrNetworkUnavailable where
                 [ "The node backend is unreachable: ", err
                 , ". Trying again in a bit might work."
                 ]
-        ErrNetworkInvalid network ->
+        ErrNetworkInvalid ntwrk ->
```

Comment:
I actually find this _**harder**_ to read! :)

Perhaps either of the following would be nicer:
* `network` (the original name); or
* `n` (if we want to choose something short)

Reply:
I had to change the name because it now clashes with the network endpoints. I am fine with `n`.